res/skins/Deere/effect_parameter_knob.xml

```diff
@@ -19,16 +19,19 @@
         <Layout>horizontal</Layout>
         <Size>me,20f</Size>
         <Children>
-          <KnobComposed>
-            <TooltipId><Variable name="TooltipId"/></TooltipId>
+          <EffectParameterKnobComposed>
             <Size>20f,20f</Size>
             <Knob>knob_small.svg</Knob>
             <MinAngle>-230</MinAngle>
             <MaxAngle>50</MaxAngle>
+            <EffectRack><Variable name="EffectRack"/></EffectRack>
+            <EffectUnit><Variable name="EffectUnit"/></EffectUnit>
+            <Effect><Variable name="Effect"/></Effect>
+            <EffectParameter><Variable name="EffectParameter"/></EffectParameter>
```

Comment:
IMO it is not a big deal because the skin XML only needs to be done right in one XML template. Also, it is consistent with the other effects widgets and shares the same code on the C++ side to get the EffectRack/EffectChainSlot/EffectSlot/EffectParameterSlot.